README.md

```diff
@@ -2,3 +2,12 @@
 
 This repository contains code for BxJS Website.  
 Primary website content is [BxJS Weekly](https://github.com/BuildingXwithJS/bxjs-weekly) podcast.
+
+## Install with Yarn
+* `npm install -g yarn`
+* `yarn global add gatsby-cli`
```

Comment:
There's no need to install gatsby-cli globally 🤔

Reply:
It was searching for gatsby cli when doing
`yarn start` it run `gatsby develop` (defined in package.json) so it was need globally :/

Reply:
that's weird, that shouldn't happen - it works fine without installing it on both of my machines 🤔